source/tutorials/gneiss.rst

```diff
@@ -73,42 +74,73 @@ The differential abundance techniques that we will be running will utilize log r
 
 .. command-block::
 
    qiime gneiss correlation-clustering \
      --i-table composition.qza \
      --o-clustering hierarchy.qza
 
-Now that we have a tree that defines our partitions, we can perform the ILR transform.  Note that the ILR transform just computes the log ratios between groups; in this case we will be computing log ratios between groups of anti-correlated features.
+
+Option 2: Gradient-clustering
+---------------------------------------------------------------
+An alternative to co-occurence clustering is to create a tree based on a numeric metadata category. With ``gradient-clustering``, we can group taxa that occur in similar ranges of a metadata category. In this example, we will create a tree (hierarchy) using the metadata category Age. Note that the metadata category can have no missing variables, and must be numeric.
 
 .. command-block::
 
-   qiime gneiss ilr-transform \
+   qiime gneiss gradient-clustering \
+     --i-table composition.qza \
+     --m-gradient-file sample-metadata.tsv \
+     --m-gradient-column Age \
+     --o-clustering hierarchy.qza
+
+An important consideration for downstream analyses is the problem of overfitting. When using ``gradient-clustering``, you are creating a tree to best highlight compositional differences along the metadata category of your choice, and it is possible to get false positives. Use ``gradient-clustering`` with caution.
+
+
+Building linear models using balances
+---------------------------------------------------------------
+Now that we have a tree that defines our partitions, we can perform the isometric log ratio (ILR) transform.  The ILR transform computes the log ratios between groups at each node in the tree.
+
+.. command-block::
+
+   qiime gneiss ilr-hierarchical \
      --i-table composition.qza \
      --i-tree hierarchy.qza \
      --o-balances balances.qza
 
-Once we have obtained a means to partition the features, we can now run linear regression on the balances.  In this module, the abundances will be converted to principal balances using the partition scheme that we defined earlier.  The linear regression that we will be running is called a `multivariate response linear regression`_, which boils down to performing a linear regression on each balance separately.
 
-We can use this to attempt to predict the microbial abundances based on environmental variables.  Running these models has multiple advantages over standard univariate regression, as it avoids many of the issues associated with overfitting, and can gain perspective about community-wide perturbations based on environmental parameters.
+Alternatively, as mentioned earlier we can use a phylogenetic tree to create our balances:
+.. command-block::
+
+   qiime gneiss ilr-phylogenetic \
+     --i-table composition.qza \
+     --i-tree rooted-tree.qza \
+     --o-balances balances.qza
```

Comment:
Just as a heads up, everything using `balances.qza` will now be based on the phylogenetic tree. If that's the goal, then there's no problem. Otherwise, you might rename the outputs so that we can differentiate them

Reply:
`phylogenetic-balances.qza` might be a good name for example